spokestack/asr/spokestack/cloud_client.py

```diff
@@ -13,17 +13,17 @@
 
 
 class CloudClient:
-    """ Spokestack client for cloud based speech to text
```

Comment:
this rename will be a breaking change, so it should be deployed as a new major release number, and you'll want to check all documentation/blog posts/example projects and upgrade them
also, these changes should be on a separate commit, which is also separate from the formatter changes

Reply:
I didn't think about the post changes. Would it be better to allow this to be imported from any level under `asr`? I think that would allow all of the old imports to work, but going forward each bottom class would need a unique name.

Reply:
it's not a big deal that we're changing/breaking things, we just need to make sure to do all the right things with versioning and documentation
it should be one way or the other, though, we don't want multiple ways to import the same thing

Reply:
I hate coming across an outdated post/repo so I agree updating is the best idea. for all of the posts we can just find and replace. At first that seemed like a bigger problem than it actually is. also having multiple aliases like we've seen in tensorflow can be really frustrating when one of them is deprecated.